### ISSUE TITLE
Looks like a pretty interesting project, what happened here? No longer being maintained?

### ISSUE THREAD

**Issue by ShashankaNataraj** [h5bp/Effeckt.css#307]

**Comment by arthurvr** [comment]
Sadly I think you're quite right on this one. At the moment Effeckt is not really actively maintained anymore or at least I think we can state its development slowed down :( But we still welcome contributions :)

**Comment by davidyezsetz** [comment]
@arthurvr I recently checked back with this repo and was wondering the same thing. I think it's quite sad since I still see a lot of devs struggling with animations.  To get things moving again I think #293 and #296 and possibly #306 would be great. After that #187 and #255 to get modularity going. Also some housekeeping in the issues would probably be in order ;) I'd be happy to put some time into the project so maybe you can give me your take on priorities.

**Comment by arthurvr** [comment]
It's awesome to hear that people are still interested in working on this project!

>  I'd be happy to put some time into the project so maybe you can give me your take on priorities.

I'll defer that to @h5bp/effeckt-css but I agree that the overall infrastructure tasks are important before we can move forward.

**Comment by davidyezsetz** [comment]
Definitely. A modular infrastructure also makes it a lot easier for people to contribute.

**Comment by ShashankaNataraj** [comment]
@arthurvr @davidyezsetz Id be willing to put in time for this project also! How do you folks suggest we get started? Can we have a checklist of things to be done to make this project more modular?

**Comment by shnrj** [comment]
@arthurvr @ShashankaNataraj @davidyezsetz I will be happily contributing as well. can you add us to the team and grant us push access so we can work on effeckt?

**Comment by arthurvr** [comment]
@shnrj Thanks for willing to contribute! 

> can you add us to the team and grant us push access so we can work on effeckt?

That's not really how it works :)

We'll be happy to add people to the team after regular, good contributions and if everybody else on the team is happy with that. Doesn't mean you can't help us, but for the time being I'd prefer you to stay sending PRs.

**Comment by ShashankaNataraj** [comment]
@shnrj Being added as a team member is lot of work, bro. You have to be acquainted with the style guide and should've made significant contributions to these projects.

I am not part of the h5bp team either, but doesn't stop me from making contributions to effeckt :) Basically fork the repo and check the issues list for things you can contribute on and make PR's for your changes. If it gets approved, fine, otherwise you know what you're doing wrong :) 

"If Im wrong, Im doubly benefitted" ;)
